.github/workflows/ci.yml

```diff
@@ -49,9 +49,6 @@ jobs:
 
             - name: Install pip deps
               run: |
-                # We need to get batoid onto conda, but for now, this is a separate step.
-                pip install batoid
-                pip install batoid_rubin
```

Comment:
Just to be clear: we don't need these because they are available from the conda distribution now?

Reply:
That's right. I added them to the standalone-requirements file.

Actually we don't need rubin-sim either.  But, I kept it this way so as to not download the parts we need.